code/gapfill4EssentialTasks.m

```diff
@@ -102,6 +102,8 @@
 modelChanges = docModelChanges(model_orig, outModel);
 
 
+end
```

Comment:
This doesn't change anything, but I believe that sub-functions should come _after_ the main function `end` statement, so I have corrected that here.

Reply:
Now this sub-function just blocks the other biomass reactions. I wonder if it's better to just delete them?

Reply:
I don't think it makes a difference, so maybe just leave it as is for now.

Reply:
ok